apps/web/src/components/DataPill.tsx

```diff
@@ -42,6 +43,7 @@ export const DataPill = ({
   }
 
   if (isStatus("error")) {
+    Sentry.captureException(query.error);
```

Comment:
Will this trigger on every re-render?

Reply:
Yes, any issue with that? At least now with 100% sampling

Reply:
`useEffect` to the rescue!